portals/knowledge-management/src/Pages/ViewArticle.vue

```diff
@@ -107,20 +105,20 @@ function getData() {
                 </div>
                 <div v-else>
                     <main class="flex flex-col gap-8">
-                        <Breadcrumbs
-                            :breadcrumbs="[
-                                { name: category.name, route: 'view-category', params: { categoryId: category.id } },
-                            ]"
-                            currentCrumb="Articles"
-                        ></Breadcrumbs>
+                        <Breadcrumbs :breadcrumbs="[
+                            { name: category.name, route: 'view-category', params: { categoryId: category.id } },
+                        ]" currentCrumb="Articles"></Breadcrumbs>
 
                         <div class="flex flex-col gap-3">
                             <div class="prose max-w-none">
                                 <h1>{{ article.name }}</h1>
+                                <div class="text-gray-500 flex items-center space-x-1 mb-4">
+                                    <ClockIcon class="h-4 w-4 flex-shrink-0" aria-hidden="true" />
+                                    <span class="text-xs">Last updated: {{ article.lastUpdated }}</span>
+                                </div>
+                                <hr class="my-4" />
```

Comment:
I would recommend using a `<div>` here with `border-t` as a `<hr>` symbolises a thematic break, which this isn't, it's a styling choice

Reply:
I agree

Reply:
Fixed.